### ISSUE TITLE
Poor performance on some classic heterophilic benchmarks

### ISSUE THREAD

**Issue by JhuoW** [hedongxiao-tju/HOG-GCN#1]
Hi,

I apply the code implementation on more classic heterophilic graphs such as Chameleon, Actor, and Squirrel, which are widely used in prior related works (H2GCN, GGCN, CPGNN, GeomGCN, etc.).  However, I can not get the desired results, sometimes even less effective than vanilla GCN and GraphSAGE. Would you mind providing the best hyperparameters of HOG-GCN on these datasets?

In my opinion, I think the strong performance on hetero graphs exhibited in the paper mainly comes from the first part of Eq.(10), i.e., Z^(l-1)W_e, and you adopt MLP to output:  https://github.com/hedongxiao-tju/HOG-GCN/blob/9e3844e106ecd36c492ec10775dbccf1fd92b20d/models_homo.py#L99 .  Because when I adopt it on GraphSAGE, it can also achieve the best results, even better than HOG-GCN.

**Comment by wt-tju** [comment]
Thank you for your attention. First, at the last days before the deadline of AAAI-2022, we train our model on Chameleon, Squirrel datasets, the performance is not satisfactory since we have no enough time to tune the hyper-parameters.  But on my opinion, the key parameters that matter are dropout and learning rate.  Second, I agree, sometimes, the first part of Eq. (10) plays a very important role, but it do not always does. Finally, if you want to further discuss the performance or if you have any other question, you can email me through "2019216113@tju.edu.cn"
